app/build.gradle

```diff
@@ -94,6 +104,29 @@ android {
   }
 }
 
+def server = new ScreenshotTestsServer(rootProject.file("docs"))
+
+tasks.create("startScreenshotsServer") {
+  doFirst {
+    server.start()
+  }
+  finalizedBy("stopScreenshotsServer")
+}
+
+tasks.create("stopScreenshotsServer") {
+  doLast {
+    server.close()
+  }
+}
+
+tasks.create("recordScreenshots") {
```

Comment:
please don't forget to update the contributing guide

Reply:
Will do

Reply:
Done